react/Newsletter.tsx

```diff
@@ -1,124 +1,149 @@
-import React, { ComponentType, PropsWithChildren, FormEvent } from 'react'
-import { useCssHandles } from 'vtex.css-handles'
+import React, { ComponentType, PropsWithChildren, FormEvent } from "react";
```

Comment:
The linting/formatting on this file is off. Please make sure to run `yarn lint --fix` in this file.

Reply:
@victorhmp done!